2022-02-03 15:03:59,556 - [WARNING] - row_handler.ObjectRowHandler - (ObjectRowHandler.py:handle:16) - Dublicate value [путь] for key [3]
2022-02-03 15:03:59,557 - [WARNING] - row_handler.ColorRowHandler - (ColorRowHandler.py:handle:16) - Dublicate value [26112] for key [зеленый]
2022-02-03 15:32:09,283 - [WARNING] - station.Station - (Station.py:fill_dict:44) - Dublicate value for key in row [['путь', '3']]
2022-02-03 15:32:09,284 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['16777215', 'белый']]
2022-02-03 15:32:09,285 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['200', 'красный']]
2022-02-03 15:32:09,285 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['255', 'красный']]
2022-02-03 15:32:09,286 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['196', 'красный']]
2022-02-03 15:32:09,286 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['0', 'черный']]
2022-02-03 15:32:09,286 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['3000365', 'зеленый']]
2022-02-03 15:32:09,287 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['2934828', 'зеленый']]
2022-02-03 15:32:09,287 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['26112', 'зеленый']]
2022-02-03 15:32:09,288 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['51200', 'зеленый']]
2022-02-03 15:32:09,289 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['16711680', 'синий']]
2022-02-03 15:32:09,291 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['65535', 'желтый']]
2022-02-03 15:32:09,298 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['4413018', 'серый']]
2022-02-03 15:32:09,299 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['5197647', 'серый']]
2022-02-03 15:32:09,300 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['6587018', 'коричневый']]
2022-02-03 15:32:09,301 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['33023', 'оранжевый']]
2022-02-03 15:32:09,301 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['16703115', 'голубой']]
2022-02-03 15:32:09,302 - [ERROR] - station.Station - (Station.py:fill_dict:42) - Error while parsing row [['26112', 'зеленый']]
2022-02-03 15:35:00,136 - [WARNING] - station.Station - (Station.py:fill_dict:44) - Dublicate value for key in row [['путь', '3']]
2022-02-03 15:35:00,137 - [WARNING] - station.Station - (Station.py:fill_dict:44) - Dublicate value for key in row [['26112', 'зеленый']]
2022-02-03 16:57:22,760 - [WARNING] - station.Station - (Station.py:fill_dict:47) - Dublicate value for key in row [['путь', '3']]
2022-02-03 16:57:22,766 - [WARNING] - station.Station - (Station.py:fill_dict:47) - Dublicate value for key in row [['26112', 'зеленый']]
2022-02-03 17:06:54,222 - [WARNING] - station.Station - (Station.py:fill_dict:47) - Dublicate value for key in the row [['путь', '3']]
2022-02-03 17:06:54,223 - [WARNING] - station.Station - (Station.py:fill_dict:47) - Dublicate value for key in the row [['26112', 'зеленый']]
2022-02-03 17:06:54,224 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the  row [['1', 'замкнут-разделка+', '3', 'серый']]
2022-02-03 17:06:54,225 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the  row [['1', 'замкнут-разделка-', '0', 'зеленый']]
2022-02-03 17:06:54,226 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the  row [['1', 'замкнут-разделка-', '2', 'серый']]
2022-02-03 17:06:54,226 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the  row [['1', 'занят-разделка+', '3', 'серый']]
2022-02-03 17:06:54,227 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the  row [['1', 'занят-разделка-', '0', 'зеленый']]
2022-02-03 17:06:54,230 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the  row [['1', 'занят-разделка-', '2', 'серый']]
2022-02-03 17:06:54,231 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the  row [['6', 'поездной', '0', 'зеленый']]
2022-02-03 17:25:35,027 - [WARNING] - station.Station - (Station.py:fill_dict:47) - Dublicate value for key in the row [путь     ; 3]
2022-02-03 17:25:35,031 - [WARNING] - station.Station - (Station.py:fill_dict:47) - Dublicate value for key in the row [26112    ; зеленый]
2022-02-03 17:25:35,037 - [ERROR] - station.Station - (Station.py:fill_dict:50) - Couldn`t handle the row [1;замкнут-разделка-;2;серый]
